web/src/layout/main.js

```diff
@@ -98,8 +104,28 @@ const Main = ({
                 ]}
                 value={electricityMixMode}
               />
+              <Toggle
+                onChange={value => dispatchApplication('timescale', value)}
+                options={[TIMESCALE.MONTHLY, TIMESCALE.LIVE].map((k) => {
+                  if (k === TIMESCALE.MONTHLY) {
+                    return { value: k, label: 'monthly historical' };
+                  }
+                  if (k === TIMESCALE.LIVE) {
+                    return { value: k, label: 'live' };
+                  }
+                  return { value: 'unknown', label: 'unknown' };
+                })}
+                value={timescale}
+              />
             </div>
             <LayerButtons />
+            {
+              timescale !== TIMESCALE.LIVE ? (
+                <div className="text-title" style={{ color: brightModeEnabled ? '#000' : '#fff' }}>
+                  {moment(currentDate).format('MMMM YYYY')}
+                </div>
```

Comment:
Something to consider design-wise is a case of connection error in the monthly view, although I'm fine living it as it is:

![Screenshot from 2020-07-13 18-23-41](https://user-images.githubusercontent.com/1216874/87328629-0d48c480-c536-11ea-98da-8d98894623bb.png)